docs/platform/concepts/tax-information.rst

```diff
@@ -27,21 +21,8 @@ According to the tax treaty between Finland and United States no tax needs to be
 Although we believe a W-8 form is not required to confirm this status, we're happy to provide a W-8BEN-E form describing our status if needed.  
 Please contact our `sales <sales@aiven.io>`_ to request one. 
 
-Updating your tax status
-------------------------
-
-Billing information can be set by Aiven users in the console. 
-
-Follow these steps:
-
-1. In the console, on the left side navigate to **Billing**. 
-2. Go to **Billing Information** section.
-3. Scroll to **Billing Profile**. Here you can update your VAT ID. 
-
-.. important::
-
-    Please be aware that you can set up Billing profile per project or create a Billing Group if you want to group costs and consolidate invoices by different business units.
-    For further information please refer to :doc:`Billing Groups </docs/platform/concepts/billing-groups>`. 
+Rest of the world
+------------------
 
-Once the billing country has been updated in the billing section, it will be used in all future invoices. 
-Billing estimates are updated approximately once an hour, so it may take up to an hour for the new tax status to become visible on the invoice estimates.
+No value-added tax is applied to services sold outside the European Union as described in Value-Added Tax Act of Finland section 69h. 
```

Comment:
Can we get a link to Value-Added Tax Act of Finland section 69h? Also, can we use "EU" instead a full form?